modules/ROOT/pages/6.3.2-release-notes.adoc

```diff
@@ -0,0 +1,42 @@
+= TinyMCE 6.3.2
+:navtitle: TinyMCE 6.3.2
+:description: Release notes for TinyMCE 6.3.2
+:keywords: releasenotes, new, changes, bugfixes
+:page-toclevels: 1
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.3.2 was released for {enterpriseversion} and {cloudname} on Wednesday, March 1^st^, 2023. It includes {productname} 6.3.2. These release notes provide an overview of the changes for {productname} 6.3.2:
+
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
```

Comment:
There isn't any change to the skins or icons.

Reply:
> There isn't any change to the skins or icons.

My understanding was that the default skin, Oxide, was re-built for every version jump.

Happy to be corrected (and its easy to remove this if this is not the case).

Reply:
That's just developer speak, sorry for the confusion. Oxide is a compiled project just like TinyMCE itself and is re-built every time TinyMCE is. That doesn't mean there are changes to document.

Github can't show you this, but if you open the `tinymce/modules/oxide` folder on the `master` branch and run `git log .` you can see nothing has changed since November. There will be changes in 6.4, that's absolutely true, but there's nothing in 6.3.2

Reply:
> Oxide is a compiled project just like TinyMCE itself and is re-built every time TinyMCE is. That doesn’t mean there are changes to document.

Noted, and removed.

> Github can't show you this, but if you open the tinymce/modules/oxide folder on the master branch and run git log . you can see nothing has changed since November. There will be changes in 6.4, that's absolutely true, but there's nothing in 6.3.2

And have turned this into a quick procedure to run as a check future Release Notes.